Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -357,8 +357,10 @@ static void updateCustomAppearance(CALayer *layer, GraphicsLayer::CustomAppearan
             playerLayer = [(WKVideoView*)layerTreeNode->uiView() playerLayer];
 #endif
         ASSERT([playerLayer respondsToSelector:@selector(setVideoGravity:)]);
-        if ([playerLayer respondsToSelector:@selector(setVideoGravity:)])
-            [(WebAVPlayerLayer*)playerLayer setVideoGravity:convertMediaPlayerToAVLayerVideoGravity(properties.videoGravity)];
+        if (RetainPtr webAVPlayerLayer = dynamic_objc_cast<WebAVPlayerLayer>(playerLayer)) {
+            if ([playerLayer respondsToSelector:@selector(setVideoGravity:)])
```

Comment:
The `respondsToSelector` is unnecessary, since we have validated that the layer is an `WebAVPlayerLayer` by this point.